content/en/docs/demos-examples/active-deployments.md

```diff
@@ -0,0 +1,39 @@
+date: 2023-09-19T15:33:17.279Z
+draft: false
+images: []
+menu:
+  docs:
+    parent: "tools-libraries"
+weight: 1110
+toc: false
+layout: matrix
+---
+
+The table below lists known active passkeys deployments which largely follow best practices and patterns. Its purpose is to provide developers with real-world examples and is not intended to be an end user facing resource.
+
+> Said differently, please don't link to this page from end user focused resources 😉
+
+To request an addition, update, or deletion, please [create an issue](https://github.com/passkeydeveloper/passkeys.dev/issues/new?assignees=&labels=&projects=&template=active-deployments.yml&title=Add+%3Cservice+name%3E).
+
+| Service Name                                      | Service Type          | Supported           | Notes                                              | Links                                                                                                                            |
+| ------------------------------------------------- | --------------------- | ------------------- | -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| [Adobe](https://www.adobe.com/)                   | Software as a Service | Web ✅<br>Apps ❓   |                                                    | [Support](https://helpx.adobe.com/manage-account/using/secure-sign-in-with-passkey.html)                                         |
+| [Afterpay](https://www.afterpay.com)              | Finance               | Web ✅<br>Apps ❓   |                                                    |                                                                                                                                  |
+| [Air New Zealand](https://www.airnewzealand.com/) | Travel                | Web ✅<br>Apps ❓   |                                                    | [Support](https://www.airnewzealand.com/cyber-security-account-protection)                                                       |
+| [Amazon.com](https://amazon.com)                  | Retail                | Web ✅<br>Apps ❓   |                                                    |                                                                                                                                  |
+| [Best Buy](https://www.bestbuy.com)               | Retail                | Web ✅<br>Apps ❓   |                                                    |                                                                                                                                  |
+| [GitHub](https://github.com)                      | Software as a Service | Web ☑️*<br>Apps ☑️* | \* beta                                            | [Support](https://docs.github.com/en/authentication/authenticating-with-a-passkey/about-passkeys)                                |
```

Comment:
I think we need some better alignment:

![Screenshot 2023-09-25 at 10 34 12 AM](https://github.com/passkeydeveloper/passkeys.dev/assets/5166470/506345c7-5277-4244-b688-7f7885241a56)


...But I can't think of how to pull that off without expanding the use of HTML in the **Supported** column specifically. Putting so much HTML into a single column in each row is not going to be easy to maintain.

Does Doks support .mdx files? My thinking is we can write a really basic React component for this column specifically that can encapsulate the use of flexbox to help with aligning the values and icons here.

Reply:
Long term this will be automatically generated from a .json file. Just trying to get something up quickly.

If you can propose some short term style changes in the next few days, please add to the PR.

Reply:
I wish the icons were properly aligned, like this:

![Screenshot 2023-09-25 at 11 25 01 AM](https://github.com/passkeydeveloper/passkeys.dev/assets/5166470/17d31d68-8bfa-4228-8513-369d255ef8a6)

But I can't pull that off without writing too much HTML to cram into an individual markdown table cell. We can fix it later.